.github/workflows/integration_test.yaml

```diff
@@ -19,6 +19,7 @@ jobs:
       self-hosted-runner-arch: X64
       self-hosted-runner-label: stg-private-endpoint
       builder-runner-label: X64
+      tmate-debug: true
```

Comment:
I'm currently not reviewing this since I'm guessing you are still debugging

Reply:
Yup will update after the image-builder PR is merged!